README.md

```diff
@@ -1,178 +1,229 @@
 
 ## Installation
 
-```bash
-npm install benchtable
+```sh
+npm i --save benchtable
 ```
 
 ## API and usage
+`BenchTable` is a class extended from the benchmark.js's [`Benchmark.Suite`](http://benchmarkjs.com/docs#Suite) class - all
+functions exposed by `Benchmark.Suite` are also provided by Benchtable and therefore a Benchtable object may be used like
+an ordinary `Benchmark.Suite`.
 
-Benchtable is a subclass of the benchmark.js's [Benchmark.Suite](http://benchmarkjs.com/docs#Suite) class - all functions exposed by `Benchmark.Suite` are also provided by Benchtable and therefore a Benchtable object may be used like an ordinary `Benchmark.Suite`.
+The `Benchmark.Suite.prototype.add` method can be used to add functions to be benchmarked, and while such functions will
+indeed be benchmarked with Benchtable–their results will not be shown in the results ASCII table.
 
-The `Benchmark.Suite.prototype.add` method can be used to add functions to be benchmarked, and while such functions will indeed be benchmarked with Benchtable -- their results will not be shown in the results ascii table.
+Please use the `BenchTable`'s `addFunction` and `addInput` methods for specifying BenchTable benchmarks, as shown below.
 
-Please use the `Benchtable.prototype.addFunction` and `Benchtable.prototype.addInput` methods for specifying Benchtable benchmarks, as shown below.
+### `BenchTable(name, options)`
+The `BenchTable` constructor. `BenchTable` is extended from [Benchmark.Suite](http://benchmarkjs.com/docs#Suite).
 
-**Constructor**
+#### Params
+- **String** `name`: A name to identify the suite.
+- **String** `options`: Options object. A special boolean option `isTransposed` is available for Benchtable–if set to `true`, the output ASCII table will be
+transposed: the benchmarked functions will be organized into columns, while
+inputs will be organized into rows. This is useful for situations where the
+number of inputs is large, and the number of functions is small. This option
+defaults to `false`.
 
-```javascript
-Benchtable(name, options);
-```
+Use the 'cycle' event to store results and build a table
+after all BenchTable functions are evaluated for all inputs.
 
-* `name` (string), `options` (object) -- [see benchmark.js API docs](http://benchmarkjs.com/docs#Suite)
-* a special boolean option `isTransposed` is available for Benchtable - if set to `true`, the output ascii table will be transposed: the benchmarked functions will be organized into columns, while inputs will be organized into rows. This is useful for situations where the number of inputs is large, and the number of function is small. This option defaults to `false`.
+### `run(config)`
+Runs the suite.
 
-**Specify functions to be benchmarked**
+#### Params
+- **Object** `config`: The options object. [See `benchmark.js` API docs.](http://benchmarkjs.com/docs#Suite_prototype_run)
 
-```javascript
-Benchtable.prototype.addFunction(name, func);
-```
+### `run(name, fun, options)`
+addFunction
```

Comment:
The jsdoc comment was wrong. Addressed in d0b1daa5dd57e2cdc8ad70abefeb894b472a7e33 and rebuilt README. :rocket: